front/app/containers/Admin/projects/project/analysis/InputPreview/LongFieldValue.tsx

```diff
@@ -117,16 +118,7 @@ const FieldValue = ({ projectId, phaseId, customFieldId, input }: Props) => {
 
   switch (customField.data.attributes.code) {
     case 'title_multiloc':
-      return (
-        <Box>
-          <Title variant="h3" my="0px">
-            <T
-              value={input.attributes[customField.data.attributes.key]}
-              supportHtml={true}
-            />
-          </Title>
-        </Box>
-      );
+      return <TitleMultilocLongField input={input} customField={customField} />;
     case 'body_multiloc':
```

Comment:
I started doing the cleanup we discussed in this PR, but made the decision to move the rest of the changes into [this other branch](https://github.com/CitizenLabDotCo/citizenlab/pull/10096) to keep the PRs easier to review :)